### ISSUE TITLE
Drop day table question

### ISSUE THREAD

**Issue by farski** [PRX/metrics.prx.org#394]
Sorry didn't come up with a good title for this

Currently if you load an episode into the DD view that's been out for fewer days than the number of selected days (eg, 60 Days is selected, and an episode has been out for 12 days), the table lists a number under 60 Day Downloads.

I'm wondering if it'd be good to have some sort of indicator on numbers that don't represent the full selected period

Or, alternatively, adding an Age column?

**Comment by sandikbarr** [comment]
what about adding an `*` to the values in the `60 Day Downloads` column with a title/hover text that reads something like "# days" for episodes that have partial data?

**Comment by farski** [comment]
Yeah that's along the lines of what I was imaging. Or we could try to come up with some consistent icon for partial data. There may be other places that could be useful.
